spec/requests/account/users_controller_spec.rb

```diff
@@ -1,16 +1,60 @@
 require 'rails_helper'
 
 RSpec.describe Account::UsersController, type: :request do
-  let(:admin) { create(:user, :admin) }
+  let(:admin)          { create(:user, :admin) }
+  let(:admin_id)       { admin.id }
 
   before do
     sign_in admin
   end
 
-  describe "users" do
+  describe 'GET #index' do
     it "returns http success if signed in as admin" do
       get '/account/users'
-      expect(response).to have_http_status(:success)
+
+      expect(response).to be_successful
+    end
+  end
+
+  describe 'GET #show' do
+    it 'returns http success status' do
+      get "/account/users/#{admin.id}"
+
+      expect(response).to be_successful
+    end
+  end
+
+  describe 'GET #edit' do
+    it 'renders edit template' do
+      get "/account/users/#{admin.id}/edit"
+
+      expect(response).to be_successful 
+    end
+  end
+
+  describe 'PATCH #update' do
+    let(:new_admin_name) { Faker::Name.first_name }
+    let(:user_params)    { { user: { first_name: new_admin_name, last_name:  'Last name Admin' } } }
```

Comment:
you can use `attributes_for :user` instead of `{ user: { first_name: new_admin_name, last_name: 'Last name Admin' } }`